db/migrate/20230425144000_create_pacman_integration.rb

```diff
@@ -0,0 +1,57 @@
+class CreatePacmanIntegration < Caseflow::Migration
+  def change
+    create_table :vbms_communication_packages do |t|
+      t.string :file_number
+      t.bigint :document_referenced, default: [], array: true
+      t.string :status
+      t.string :comm_package_name, null: false
+      t.timestamps
+
+      t.references :vbms_uploaded_document, index: true, foreign_key: { to_table: :vbms_uploaded_documents }
+
+      t.references :created_by , index: true, foreign_key: { to_table: :users }
+      t.references :updated_by, index: true, foreign_key: { to_table: :users }
+    end
+
+    create_table :vbms_distributions do |t|
+      t.string :distribution_type, null: false
```

Comment:
```suggestion
      t.string :recipient_type, null: false
```

After taking another look at the documentation, it seems like `recipient_type` may be a better name for this column.